### PR TITLE
Slurm train scripts

### PR DESCRIPTION
**Pull request by rayleizhu**

### CODE REVIEW

File: detrex/config/configs [hunk 1]
```diff
@@ -0,0 +1 @@
+/mnt/lustre/zhulei1/GitRepo/detrex/configs
```
Comment (rentainhe) [inline detrex/config/configs:1]: Hi~ this seems like a specific submodule linked to your local projects, if this is needed or not~

**Comment by rayleizhu** [comment]
No, it is generated by your setup scripts, not by mine. I forget to add it to .gitignore.

**Comment by rentainhe** [comment]
Maybe you can try to add it to gitignore or just remove this file (not add it by `git add`)~ the other parts are OK~ @rayleizhu

**Comment by rayleizhu** [comment]
> Maybe you can try to add it to gitignore or just remove this file (not add it by `git add`)~ the other parts are OK~ @rayleizhu

Done.

**Review by rentainhe — APPROVED** [review]

**Comment by rentainhe** [comment]
> > Maybe you can try to add it to gitignore or just remove this file (not add it by `git add`)~ the other parts are OK~ @rayleizhu
> 
> Done.

Merged~ Thanks a lot for your contribution~

**Comment by rentainhe** [comment]
BTW, would you like to provide some tutorial about how to use the slurm script with a new pull request~ @rayleizhu , maybe create a new markdown file under tools~

**Comment by rayleizhu** [comment]
> BTW, would you like to provide some tutorial about how to use the slurm script with a new pull request~ @rayleizhu , maybe create a new markdown file under tools~

Fine. Maybe by next Tuesday.

**Pull request merged**